src/device.c

```diff
@@ -302,6 +302,11 @@ static int create_data_item_polling(void)
 	return rc;
 }
 
+void set_knot_cloud_log_priority(int priority)
```

Comment:
Use the prefix device_ and make it generic.
Use something like device_set_log_priority(int priority);